projects/canopy/src/lib/forms/radio/radio-button.component.html

```diff
@@ -6,10 +6,14 @@
   [attr.name]="name"
   [attr.value]="value"
   [attr.aria-invalid]="errorClass"
+  [attr.aria-describedby]="ariaDescribedBy || null"
   class="lg-radio-button__input"
   type="radio"
 />
 
 <label [attr.for]="id" class="lg-radio-button__label">
   <ng-content></ng-content>
 </label>
+<div class="lg-radio-button__content">
```

Comment:
Could you look at doing something like in this answer please? https://stackoverflow.com/questions/35107211/how-to-check-whether-ng-content-is-empty-in-angular-2-till-now

There is no need for the content wrapper if the hint is not added. Also could it be called `lg-radio-button__hint`?

Reply:
Renamed the class & and placed conditional render logic on the div wrapper.